src/Core/Framework/Api/OAuth/ClientRepository.php

```diff
@@ -49,6 +49,10 @@ public function validateClient($clientIdentifier, $clientSecret, $grantType): bo
             return true;
         }
 
+        if ($grantType === 'test_grant') {
```

Comment:
This doesn't look right. Does it check against the new grant type? Can you use the constant of the new grant type here?

Reply:
This was testing code.. It can be removed completely